SPECS/sudo/sudo.spec

```diff
@@ -36,6 +37,8 @@ the ability to run some (or all) commands as root or another user while logging
     --docdir=%{_docdir}/%{name}-%{version} \
     --with-env-editor \
     --with-pam \
+    --with-ldap \
+    --with-ldap-conf-file=/etc/sudo-ldap.conf \
```

Comment:
is there a reason/need for defaulting to a separate ldap conf file here? Or just copying how Debian/Ubuntu does it?

Reply:
I essentially followed what was on the man page [here](https://linux.die.net/man/5/sudoers.ldap).

Reply:
The man page also mentions this
> Also note that on systems using the OpenLDAP libraries, default values specified in /etc/openldap/ldap.conf or the user's .ldaprc files are not used.
so I think leaving it to the current path is fine.

Reply:
There seems to be a slight difference between the man page you linked and the official documentation: https://www.sudo.ws/docs/man/1.9.13/sudoers.ldap.man/#Configuring_ldap.conf. The default value is `/etc/ldap.conf` and it's shared among other ldap clients. Unless users have specific requirements for using sudo with ldap, they don't need to create a separate config file. Let's remove that parameter and leave it to default to `/etc/ldap.conf`

Reply:
Sounds good.